net-p2p/deluge/deluge-2.1.1.ebuild

```
@@ -0,0 +1,133 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="8"
+
+PYTHON_COMPAT=( python3_{8..10} )
+DISTUTILS_USE_PEP517=setuptools
+DISTUTILS_SINGLE_IMPL=1
+inherit xdg distutils-r1 systemd
+
+DESCRIPTION="BitTorrent client with a client/server model"
+HOMEPAGE="https://deluge-torrent.org/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://git.deluge-torrent.org/${PN}"
+else
+	SRC_URI="http://download.deluge-torrent.org/source/$(ver_cut 1-2)/${P}.tar.xz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="console gui libnotify sound webinterface"
+REQUIRED_USE="
+	${PYTHON_REQUIRED_USE}
+	libnotify? ( gui )
+	sound? ( gui )
+"
```

[I wasn't correct about `RDEPEND="${PYTHON_DEPS}"`](https://projects.gentoo.org/python/guide/basic.html#python-deps-and-python-required-use):
```
This does not apply to distutils-r1 as it does the above assignment by default.
```
We need to remove it. I'm sorry.
```suggestion
REQUIRED_USE="
	libnotify? ( gui )
	sound? ( gui )
"
```

---

Not a problem!